modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1545,10 +1545,23 @@ impl MintClientModule {
             )
             .await
             .context(ReissueExternalNotesError::AlreadyReissued)?;
+
         let mut dbtx = self.client_ctx.module_db().begin_transaction().await;
+
         self.client_ctx
             .log_event(&mut dbtx, OOBNotesReissued { amount })
             .await;
+
+        self.client_ctx
+            .log_event(
+                &mut dbtx,
+                ReceivePaymentEvent {
+                    operation_id,
+                    amount,
+                },
+            )
+            .await;
```

Comment:
Could we just expand the existing event? Also the same crash tolerance correctness issue here (pre-existing, but still, let's do it correctly this time around).

Reply:
Discussed oob: the naming being consistent is a feature. We should discuss how we might phase out the old events @dpc.